docs/configuration/index.md

```diff
@@ -1197,79 +1162,15 @@ The `druid.supervisor.idleConfig.*` specified in the Overlord runtime properties
 
 #### Overlord dynamic configuration
 
-The Overlord can be dynamically configured to specify how tasks are assigned to workers.
-
-The JSON object can be submitted to the Overlord via a POST request at:
-
-```
-http://<OVERLORD_IP>:<port>/druid/indexer/v1/worker
-```
-
-Optional header parameters for auditing the config change can also be specified.
+The Overlord has dynamic configurations to tune how Druid assigns tasks to workers.
+You can configure these parameters using the [web console](../operations/web-console.md) or through the [Overlord dynamic configuration API](../api-reference/dynamic-configuration-api.md#overlord-dynamic-configuration).
 
-|Header Param Name|Description|Default|
-|-----------------|-----------|-------|
-|`X-Druid-Author`| author making the config change|""|
-|`X-Druid-Comment`| comment describing the change being done|""|
-
-An example Overlord dynamic config is shown below:
```

Comment:
What's the reasoning behidn wanting to remove this example completely vs collapsing it in a details block?

Reply:
We already show a simplified example of the Overlord dynamic config in the[Dynamic configuration API](https://github.com/apache/druid/blob/master/docs/api-reference/dynamic-configuration-api.md#sample-response-3)  document. I guess it doesn't hurt to leave this example here (but put it in a details block) since it shows the `autoScaler` object.